### ISSUE TITLE
Can't compile for iOS 7.1

### ISSUE THREAD

**Issue by pieisgood** [mxcl/PromiseKit#157]
I Can't seem to get this package to compile for iOS 7.1, it seems that OMGHTTPURLRQ relies on 8.0 features? Am I missing something here?

**Comment by mxcl** [comment]
If you don't use CocoaPods or Carthage then you will have to make manual changes to various xcodeproj settings to compile it in the way you want.

We support only officially support CocoaPods and Carthage. If you chose to do it without these tools then you are saying “Hi, I know what I am doing”, thus because it is impossible to compile a framework for iOS 7, we have to set it to 8.0.

If you thus want to compile for 7.x you will have to go into the projects manually and fix them to not compile as 8.0 frameworks.

Basically: use a dependency management tool.

**Comment by mxcl** [comment]
Change the deployment target to iOS 7.0.

It'll build, but it won't be a real 8.0 framework so you will have to manually force linkage.

We only provide support for PromiseKit with Carthage and CocoaPods. You *can* use PromiseKit without those tools, but you will have problems like this. This is unfortunate and simply because the iOS 7 to iOS 8 transition has been difficult.
